src/BlockEntityRenderers/ChurchBellPartRenderer.cs

```diff
@@ -0,0 +1,81 @@
+﻿using RPVoiceChat.BlockEntities;
+using Vintagestory.API.Client;
+using Vintagestory.API.MathTools;
+
+namespace RPVoiceChat.BlockEntityRenderers
+{
+    public class ChurchBellPartRenderer : IRenderer
```

Comment:
Most if not all of this class logic is already part of ChurchBellLayerRenderer, it shouldn't be duplicated here.